python/cudnn/flex_attention/_compat/layout_utils.py

```
@@ -0,0 +1,86 @@
+def reshape_acc_to_mn(acc: cute.Tensor, transpose: bool = False) -> cute.Tensor:
+    return cute.make_tensor(acc.iterator, convert_layout_acc_mn(acc.layout, transpose=transpose))
+
+
+@cute.jit
+def convert_layout_acc_frgA(acc_layout: cute.Layout) -> cute.Layout:
+    # For back-to-back GEMMs, convert the first accumulator to the fragment-A layout.
+    # For SM90 FP16/BF16, convert ((2, 2, N / 8), MMA_M, MMA_N) to ((2, 2, 2), MMA_M, (N / 16, MMA_N)).
+    # If N / 8 is odd, we'll convert to ((2, 2, 1), MMA_M, N / 8, MMA_N).
+    assert cute.rank(acc_layout.shape[0]) == 3
+    div = 2 if const_expr(acc_layout.shape[0][2] % 2 == 0) else 1
+    l = cute.logical_divide(acc_layout, ((None, None, div), None, None))  # ((2, 2, (2, N / 16)), MMA_M, MMA_N)
```

Ruff (0.16.2)

[error] Ambiguous variable name: `l`

(E741)